exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -469,9 +423,69 @@ func (b *Bitfinex) wsHandleData(respRaw []byte) error {
 	return nil
 }
 
-func (b *Bitfinex) handleWSChannelUpdate(c *stream.ChannelSubscription, chanID int, eventType string, d []interface{}) error {
+func (b *Bitfinex) handleWSEvent(respRaw []byte) error {
+	event, err := jsonparser.GetUnsafeString(respRaw, "event")
+	if err != nil {
+		return fmt.Errorf("error parsing WS event name: %w from message: %s", err, respRaw)
+	}
+	switch event {
+	case wsEventSubscribed:
+		subID, err := jsonparser.GetUnsafeString(respRaw, "subId")
+		if err != nil {
+			return fmt.Errorf("error parsing WS subscribed event subId: %w from message: %s", err, respRaw)
+		}
+		if !b.Websocket.Match.IncomingWithData("subscribe:"+subID, respRaw) {
+			return fmt.Errorf("%v channel subscribe listener not found", subID)
+		}
+	case wsEventUnsubscribed:
+		chanID, err := jsonparser.GetUnsafeString(respRaw, "chanId")
+		if err != nil {
+			return fmt.Errorf("error parsing WS unsubscribed event chanId: %w from message: %s", err, respRaw)
+		}
+		if !b.Websocket.Match.IncomingWithData("unsubscribe:"+chanID, respRaw) {
+			return fmt.Errorf("%v channel unsubscribe listener not found", chanID)
+		}
+	case wsEventError:
+		if subID, err := jsonparser.GetUnsafeString(respRaw, "subId"); err == nil {
+			if !b.Websocket.Match.IncomingWithData("subscribe:"+subID, respRaw) {
+				return fmt.Errorf("%v channel subscribe listener not found", subID)
+			}
+		} else if chanID, err := jsonparser.GetUnsafeString(respRaw, "chanId"); err == nil {
+			if !b.Websocket.Match.IncomingWithData("unsubscribe:"+chanID, respRaw) {
+				return fmt.Errorf("%v channel unsubscribe listener not found", chanID)
+			}
+		} else {
+			return fmt.Errorf("unknown channel error; Message: %s", respRaw)
+		}
+	case wsEventAuth:
+		status, err := jsonparser.GetUnsafeString(respRaw, "status")
+		if err != nil {
+			return fmt.Errorf("error parsing WS auth event status: %w from message: %s", err, respRaw)
+		}
+		if status == "OK" {
+			var glob map[string]interface{}
+			if err := json.Unmarshal(respRaw, &glob); err != nil {
+				return fmt.Errorf("unable to Unmarshal auth resp; Error: %w Msg: %v", err, respRaw)
+			}
+			// TODO - Send a better value down the channel
+			b.Websocket.DataHandler <- glob
+		} else {
+			errCode, err := jsonparser.GetInt(respRaw, "code")
+			if err != nil {
+				log.Errorf(log.ExchangeSys, "%s error parsing WS auth event error code: %s", b.Name, err)
+			}
+			return fmt.Errorf("WS auth subscription error; Status: %s Error Code: %d", status, errCode)
+		}
+	default:
+		return fmt.Errorf("unknown WS event msg: %s", respRaw)
```

Comment:
```
[ERROR] | WEBSOCKET | 04/10/2023 14:19:58 | exchange Bitfinex websocket error - unknown WS event msg: {"event":"info","version":2,"serverId":"854ce848-d3d1-46a4-ad89-378ef895de60","platform":{"status":1}}
[ERROR] | WEBSOCKET | 04/10/2023 14:19:58 | exchange Bitfinex websocket error - unknown WS event msg: {"event":"conf","status":"OK","flags":196608}
```

Reply:
Ah - Yep - I wanted this in here to handle the messages we were ignoring.

Reply:
Yeah happy with leaving the error in. An error sink of anything unhandled is way better in my opinion.

Reply:
Fixed: 872499d1